core/manifest/ConfigStaticManifest.php

```diff
@@ -183,17 +183,15 @@ function getStatics() {
 	}
 
 	/**
-	 * Get the next token to process, incrementing the pointer
+	 * Get the next token to process, incrementing the pointer, skips any whitespace tokens
 	 *
-	 * @param bool $ignoreWhitespace - if true will skip any whitespace tokens & only return non-whitespace ones
 	 * @return null | mixed - Either the next token or null if there isn't one
 	 */
-	protected function next($ignoreWhitespace = true) {
```

Comment:
I don’t think we can merge this particular change in to 3. It’s _unlikely_ that anyone is extending the parser, but it’s still possible - and this change could be potentially breaking even if we leave the parameter there (what if someone has subclassed the parser, and is using `->next(false)` in their own code?).